src/app/notes/[noteId]/NoteBox.tsx

```diff
@@ -2,44 +2,101 @@
 import { Note } from '@prisma/client';
 import { toast } from 'react-toastify';
 import { updateNote } from '@/app/notes/actions';
+import { Badge, Button, Input, Tooltip } from 'react-daisyui';
+import React, { useState } from 'react';
+import { Edit, Save } from 'react-feather';
+import MDEditor from '@uiw/react-md-editor';
+import Markdown from 'react-markdown';
 
 export const NoteBox = ({ note }: { note: Note }) => {
-  const saveNote = async (form: FormData) => {
+  const [name, setName] = useState(note.name);
+  const [noteMD, setNoteMD] = useState<string | undefined>(note.text);
```

Comment:
```suggestion
  const [noteMD, setNoteMD] = useState<string | undefined>(note.text || '\n');
```
this is annyoing, but it will fix the editor not being high enough to comfortably start editing when the note is empty. I fucked around with css but alas, no luck